docs/docs/troubleshooting.mdx

```diff
@@ -200,3 +221,41 @@ First, you will need to create new node groups, one for each type of GPU instanc
 
 Second, you will need to create a new JupyterLab profile to select the GPU node.
 In this way, you'll have a separate profile for GPUs which users can select.
+
+### Migrate existing environments from a namespaces to another
+
+When the user creates a new environment, it requires selecting a namespace. This namespace is used to identify the group permissions to
+that environment and the location where the such environment needs to be stored.
+To migrate environments from one namespace to another, the user must manually build all environments in the new namespace.
+Once migrated, the user can delete the old namespace, permanently deleting all existing environments built against it.
+
+### Handle Access restricted namespaces
+
+The built in namespace (`nebari-git`) is used to store the environments contained in the nebari-config.yaml file.
+As this namespace is controlled by the Nebari conda-store server and not the user, it is not possible to modify
+or delete it without proper permissions.
+
+To modify this namespace, the user need to manually update the `conda-store-config.py` configmap located in the
```

Comment:
Similar question to earlier, do the administrators need to do this or individual users?